sway/handlers.c

```diff
@@ -172,6 +173,21 @@ static bool handle_view_created(wlc_handle handle) {
 		set_focused_container(newview);
 		swayc_t *output = swayc_parent_by_type(newview, C_OUTPUT);
 		arrange_windows(output, -1, -1);
+		// check if it matches for_window in config and execute if so
+		list_t *criterias = criterias_for(newview);
+		for (int i = 0; i < criterias->length; i++) {
+			struct criteria *crit = criterias->items[i];
+			sway_log(L_DEBUG, "for_window '%s' matches new view %p, cmd: '%s'",
+					crit->crit_raw, newview, crit->cmdlist);
+			struct cmd_results *res = handle_command(crit->cmdlist);
```

Comment:
We need to ensure that the new window receives focus before executing the command.